packages/components/dialog/src/dialog.scss

```diff
@@ -5,26 +5,36 @@ $md-breakpoint: 600px;
   --_border: var(--sl-border-width-dialog-default) solid var(--sl-color-dialog-border);
   --_border-radius-mobile: var(--sl-border-radius-dialog-mobile);
   --_border-radius-desktop: var(--sl-border-radius-dialog-desktop);
+  --_border-radius: var(--_border-radius-mobile);
   --_header-gap: var(--sl-space-dialog-gap-header);
   --_heading-direction: var(--sl-dialog-header-flex-direction);
   --_heading-gap: var(--sl-space-dialog-gap-header);
   --_body-gap-mobile: var(--sl-space-dialog-mobile-body-gap);
   --_body-gap-desktop: var(--sl-space-dialog-desktop-body-gap);
-  --_footer-direction: var(--sl-dialog-footer-flex-direction);
+  --_body-gap: var(--_body-gap-mobile);
+  --_footer-direction-mobile: column-reverse;
+  --_footer-direction-desktop: var(--sl-dialog-footer-flex-direction);
+  --_footer-direction: var(--_footer-direction-mobile);
   --_footer-align: var(--sl-dialog-footer-justify-content);
   --_max-inline-size-mobile: var(--sl-size-dialog-full);
-  --_min-inline-size-desktop: var(--sl-size-dialog-min);
+  --_min-inline-size: var(--sl-size-dialog-min);
   --_max-inline-size-desktop: var(--sl-size-dialog-max);
+  --_max-inline-size: var(--_max-inline-size-mobile);
   --_padding-block-mobile: var(--sl-space-dialog-mobile-body-block);
   --_padding-inline-mobile: var(--sl-space-dialog-mobile-body-inline);
   --_padding-block-desktop: var(--sl-space-dialog-desktop-body-block);
   --_padding-inline-desktop: var(--sl-space-dialog-desktop-body-inline);
+  --_padding-block: var(--_padding-block-mobile);
```

Comment:
i meant actually remove all the specific mobile and desktop variables, just use the token directly.